features/doctrine/boolean_filter.feature

```diff
@@ -166,6 +166,210 @@ Feature: Boolean filter on collections
     """
     And the JSON node "hydra:totalItems" should be equal to 10
 
+  Scenario: Get collection by embeddedDummy.dummyBoolean true
+    Given there is "15" embedded dummy objects with embeddedDummy.dummyBoolean true
+    And there is "10" embedded dummy objects with embeddedDummy.dummyBoolean false
+    When I send a "GET" request to "/embedded_dummies?embeddedDummy.dummyBoolean=true"
+    Then the response status code should be 200
+    And the response should be in JSON
+    And the header "Content-Type" should be equal to "application/ld+json; charset=utf-8"
+    And the JSON should be valid according to this schema:
+    """
+    {
+      "type": "object",
+      "properties": {
+        "@context": {"pattern": "^/contexts/EmbeddedDummy$"},
+        "@id": {"pattern": "^/embedded_dummies$"},
+        "@type": {"pattern": "^hydra:Collection$"},
+        "hydra:member": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "@id": {
+                "oneOf": [
+                  {"pattern": "^/embedded_dummies/1$"},
+                  {"pattern": "^/embedded_dummies/2$"},
+                  {"pattern": "^/embedded_dummies/3$"}
+                ]
+              }
+            }
+          }
+        },
+        "hydra:view": {
+          "type": "object",
+          "properties": {
+            "@id": {"pattern": "^/embedded_dummies\\?embeddedDummy.dummyBoolean=true"},
```

Comment:
Not a big deal but I guess the strict pattern would be `"^/embedded_dummies\\?embeddedDummy\\.dummyBoolean=true"` here and in other places.
Not sure it is worth updating it though, it's quite unlikely to be biaised